src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -572,21 +572,6 @@ export default {
 				);
 			}
 		},
-		initializeRecommendedRowAlgoExp() {
```

Comment:
this function is not referenced anywhere else

Reply:
We actually had a ticket for this but I was unclear about  it's usage... https://kiva.atlassian.net/browse/VUE-972

Seems like any code that used this must have already been removed???